modules/registry-configuring-registry-storage-install-rhodf.adoc

```diff
@@ -0,0 +1,292 @@
+  namespace: openshift-storage
+spec:
+  targetNamespaces:
+  - openshift-storage
+EOF
+----
+
+. Create your subscription. For example:
++
+[source,terminal]
+----
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: odf-operator
+  namespace: openshift-storage
+spec:
+  channel: stable-4.10
+  installPlanApproval: Automatic
+  name: odf-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
++
+[source,terminal]
+----
+$ oc get og
+----
+.Example output
++
+[source,terminal]
+----
+NAME                        AGE
+openshift-storage-xiuwang   6m38s
+$oc get ip
+NAME            CSV                               APPROVAL    APPROVED
+install-5lgrk   odf-csi-addons-operator.v4.10.1   Automatic   true
+install-xk5l2   mcg-operator.v4.10.1              Automatic   true
+$oc get sub
+NAME                                                              PACKAGE                   SOURCE             CHANNEL
+mcg-operator-stable-4.10-redhat-operators-openshift-marketplace   mcg-operator              redhat-operators   stable-4.10
+ocs-operator-stable-4.10-redhat-operators-openshift-marketplace   ocs-operator              redhat-operators   stable-4.10
+odf-csi-addons-operator                                           odf-csi-addons-operator   redhat-operators   stable-4.10
+odf-operator                                                      odf-operator              redhat-operators   stable-4.10
+$oc get csv
+NAME                              DISPLAY                       VERSION   REPLACES                          PHASE
+mcg-operator.v4.10.1              NooBaa Operator               4.10.1    mcg-operator.v4.10.0              Succeeded
+ocs-operator.v4.10.1              OpenShift Container Storage   4.10.1    ocs-operator.v4.10.0              Succeeded
+odf-csi-addons-operator.v4.10.1   CSI Addons                    4.10.1    odf-csi-addons-operator.v4.10.0   Succeeded
+odf-operator.v4.10.1              OpenShift Data Foundation     4.10.1    odf-operator.v4.10.0              Succeeded
+----
+
+. Check {rh-storage}-related Operators that are running by entering the following command
++
+[source,terminal]
+----
+$ oc get pods
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                               READY   STATUS    RESTARTS   AGE
+csi-addons-controller-manager-8444c56f7c-slwr2     2/2     Running   0          4m28s
+noobaa-operator-cfbf8d975-8dft7                    1/1     Running   0          5m50s
+ocs-metrics-exporter-6db8d7658d-kr6pw              1/1     Running   0          5m41s
+ocs-operator-6d5c5dcfdf-x44hf                      1/1     Running   0          5m41s
+odf-console-5b9889456b-555pq                       1/1     Running   0          5m40s
+odf-operator-controller-manager-865d4b4d75-v6v6z   2/2     Running   0          5m40s
+rook-ceph-operator-588867c665-dvw47                1/1     Running   0          5m41s
+----
```

Comment:
Isn't it covered by the prerequisite:

> Install the {rh-storage-first} Operator to provide object storage and `cephfs` file storage.

?

Reply:
Updated the prereqs, thanks!